src/constants/theme.js

```diff
@@ -0,0 +1,61 @@
+import { Dimensions } from "react-native";
+const { width, height } = Dimensions.get("window");
+
+export const COLORS = {
+    // base colors
+    primary: "#465BFE", // Blue
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file.